src/main/java/frameworksdrivers/DatabaseDriver.java

```diff
@@ -4,13 +4,15 @@
 
 /**
  * Driver that connects to the databases and creates the gateways. Each gateway has a reference to the connection so
- * that they are all able to communicate with the database.
+ * that they are all able to communicate with the database. The connection to the database is a static variable to
+ * ensure that the database driver only ever connects once to the database.
  */
 public class DatabaseDriver implements Driver {
+    private static final DbConnection dbConnection = new DbConnection();
```

Comment:
I added the static variable at the top of the class because this one is the most important variable for this class and I think it should stand out the most by adding it right below the class header

Reply:
This looks good and will allow for a cleaner main method